include/swift/Frontend/Frontend.h

```diff
@@ -341,11 +352,16 @@ class CompilerInstance {
 
   enum : unsigned { NO_SUCH_BUFFER = ~0U };
   unsigned MainBufferID = NO_SUCH_BUFFER;
+
+  /// Corresponds to PrimaryInput
```

Comment:
Doc comments should always end in a period, and should usually be complete sentences. (It's okay to drop the subject when it's obviously "This", though.)

Reply:
Fixed.